libraries/WiFi/src/WiFiGeneric.cpp

```diff
@@ -664,15 +664,6 @@ bool wifiLowLevelInit(bool persistent){
 
         wifi_init_config_t cfg = WIFI_INIT_CONFIG_DEFAULT();
 
-	if(!WiFiGenericClass::useStaticBuffers()) {
```

Comment:
this was added to reduce the memory footprint of the WiFi driver as it increased considerably from IDF v3 to IDF v4.

Refer to https://github.com/espressif/arduino-esp32/pull/5791 for this. @SuGlider fyi...

Reply:
Yes... Default now is WiFi Dynamic Allocation in order to reduce HEAP consumption.
If the user wants to go with Static Allocation, the application shall use `WiFiGenericClass::useStaticBuffers(true);` before setting the WiFi Mode with `WiFiGenericClass::mode()`

Reply:
@SuGlider Although the customers can call `WiFiGenericClass::useStaticBuffers(true)` to solve the problem, but I think we also need to change the value `cfg.cache_tx_buf_num = 1` to at least `4`, this will ensure the customers can send several packets at the same time.

Reply:
OK, Agree. Please commit this change. This will cost 5K RAM in the heap. It is worth it.